webots.yaml

```
@@ -0,0 +1,6 @@
+animation:
+  worlds:
+    - file: worlds/tutorial_6.wbt
+      duration: 5
+    - file: worlds/tutorial_1.wbt
+      duration: 10
```

I would swap these durations otherwise the collision avoidance animation doesn't show anything interesting and the go forward simulation is too long.

```suggestion
    - file: worlds/tutorial_1.wbt
      duration: 5
    - file: worlds/tutorial_6.wbt
      duration: 10
```